framework/tlvf/tlvf.py

```diff
@@ -287,6 +287,16 @@ def getConstValue(name, value_const):
             ret_val = hex(value_const) if str(value_const).isdigit() else value_const
             ret_err = False
         return [ret_val, ret_err]
+    
+    @staticmethod
+    def getFormattedValue(param_value):
+        try:
+            retVal = hex(param_value)
+        except:
```

Comment:
I would also like a comment here, because the logic is not so transparent.
```python
            # If it's not an integer, assume it's an enum value or other constant that can be used directly as a string.
```

Reply:
fixed